### PR TITLE
Setup/setup

### PR DESCRIPTION
**Pull request by Ant-Shell**
# Pull Request

## Description:
- Setup Vite
- Setup Vue.js
- Setup Tailwindcss

## Next Steps:
- Start building in Vue.js!

## Any other comments or questions:
N/A

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { createApp } from 'vue'
+import './style.css'
+import App from './App.vue'
+
+createApp(App).mount('#app')
```
Comment (stephanieguzm) [inline src/main.ts:5]: Is `mount()` Vue's equivalent of React's `render()` method? That's a pretty cool implementation!

File: src/App.vue [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<script setup lang="ts">
+import HelloWorld from './components/HelloWorld.vue'
+</script>
+
+<template>
```
Comment (stephanieguzm) [inline src/App.vue:5]: I haven't seen `template` used before. Assuming that it's part of Vue's implementation, what's the benefit of using it over a `<div>` or other HTML tag?

File: src/main.ts (hunk 1 above)
Comment (Ant-Shell) [reply src/main.ts:5]: I think in this case, `createApp().mount()` in Vue is closer to `ReactDOM.createRoot()` in React

More info [here](https://vuejs.org/api/application.html#createapp) and [here](https://vuejs.org/api/application.html#app-mount) if interested

File: src/App.vue (hunk 2 above)
Comment (Ant-Shell) [reply src/App.vue:5]: The `<template>` tag is indeed part of Vue's implementation. It basically encapsulates the HTML that will be rendered and does a fair amount of abstraction.

More info [here](https://vuejs.org/guide/introduction.html#single-file-components), [here](https://vuejs.org/guide/essentials/template-syntax.html#template-syntax) and [here](https://vuejs.org/guide/essentials/template-refs.html) if interested.

**Pull request merged**